### ISSUE TITLE
feat: Allow only SSO account creation

### ISSUE THREAD

**Issue by backmask** [langfuse/langfuse#509]
### Describe the feature you'd like to request

I would like to allow accounts to be created only through SSO in order to be sure only trusted users have access to langfuse

### Describe the solution you'd like to see

I would imagine an env variable like `NEXT_PUBLIC_SIGN_UP_DISABLED`, so maybe `SIGN_UP_SSO_ONLY`

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
@backmask agree, I'd suggest: `AUTH_ALLOWED_SSO_DOMAINS_ONLY=true` in combination with the existing `AUTH_DOMAINS_WITH_SSO_ENFORCEMENT`

**Comment by marcklingen** [comment]
https://github.com/langfuse/langfuse/pull/510 implemented you suggestion

Open for contributions on `AUTH_ALLOWED_SSO_DOMAINS_ONLY=true`, closing this for now
